packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -169,6 +184,14 @@ const VirtualListBaseFactory = (type) => {
 			 */
 			getComponentProps: PropTypes.func,
 
+			/**
+			 * The array for individually sized items.
+			 *
+			 * @type {Number[]}
+			 * @private
+			 */
+			itemSizes: PropTypes.array,
```

Comment:
I forgot to push the commit replacing PropTypes.array with PropTypes.arrayOf(PropTypes.number).

Reply:
I updated it